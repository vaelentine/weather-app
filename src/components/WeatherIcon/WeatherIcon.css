.weatherIcon {
    padding: 10px;
}

.iconImage {
    width: 128px;
}
