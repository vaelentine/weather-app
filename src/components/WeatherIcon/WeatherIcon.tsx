import React from 'react'
import './WeatherIcon.css'

interface iconProps {
  iconProps:any
}
const WeatherIcon:React.FC<iconProps> = ({ iconProps }) => {
  const weatherIconURL = "https://openweathermap.org/img/wn/";
  const srcUrl = `${weatherIconURL}${iconProps.icon}@2x.png`
  return (
    <div className="weatherIcon">
      <img className="iconImage" src={srcUrl} alt={iconProps.main}/>
    </div>
  )
}

export default WeatherIcon
