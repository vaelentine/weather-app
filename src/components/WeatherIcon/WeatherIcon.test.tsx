import React from "react";
import { WeatherIcon } from "./WeatherIcon";

describe('WeatherIcon', () => {
    it('should render the correct img', () => {

    })

}) 