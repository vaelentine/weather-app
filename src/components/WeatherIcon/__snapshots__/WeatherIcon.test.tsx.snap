// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherIcon renders correctly 1`] = `
<DocumentFragment>
  <div
    class="weatherIcon"
  >
    <img
      class="iconImage"
      src="https://openweathermap.org/img/wn/undefined@2x.png"
    />
  </div>
</DocumentFragment>
`;
